include/deal.II/non_matching/immersed_surface_quadrature.h

```diff
@@ -82,9 +82,10 @@ namespace NonMatching
    * $\hat{F}$, reads
    * @f[
    * \int_{S\cap F} f dr =
-   * \int_{S\cap F} f |d\bar{r}| =
-   * \int_{\hat{S}\cap\hat{F}} f \circ F_{K} | J d\hat{r}|
-   * \approx \sum_{q} f \left(F_{K}(\hat{x}_{q}) \right) |J_q \hat{t}_q| w_q,
+   * \int_{S\cap F} f \|d\bar{r}\| =
```

Comment:
That's not quite right -- `dr` is an area or volume element, and as such is a scalar. Depending on how you define the transformation, it may be positive or negative, and so using `|...|` makes sense. But it isn't a vector.

Reply:
Sorry for not fixing this sooner... No, I really do mean the vector here, but I realized that my notation was strange, so I rewrote this. Updated now.